File-Formats/PDF/Convert-HTML-To-PDF/aws.md

```diff
@@ -7,132 +7,127 @@ documentation: UG
 keywords: Assemblies
 ---
 
-The Syncfusion [HTML to PDF converter](https://www.syncfusion.com/pdf-framework/net/html-to-pdf) is a .NET library for converting webpages, SVG, MHTML, and HTML to PDF using C#. It is reliable and accurate. Using this library, you can convert HTML to PDF in AWS using Blink.
+# Convert HTML to PDF file in AWS
 
-# Setting up the AWS Toolkit for Visual Studio:
+The Syncfusion [HTML to PDF converter](https://www.syncfusion.com/pdf-framework/net/html-to-pdf) is a .NET library for converting webpages, SVG, MHTML, and HTML to PDF using C#. The result preserves all graphics, images, text, fonts, and the layout of the original HTML document or webpage. Using this library, you can convert HTML to PDF using C# with Blink rendering engine in AWS.
 
-You can create an AWS account by referring to this [link.](https://aws.amazon.com/) 
-Download and install the AWS Toolkit for Visual Studio, you can download the AWS toolkit from this [link](https://aws.amazon.com/visualstudio/). The Toolkit can be installed from Tools/Extension and updates options in Visual Studio. 
+## Setting up the AWS Toolkit for Visual Studio
 
-## Refer to the following steps to convert HTML to PDF in AWS Lambda:
+* You can create an AWS account by referring to this [link.](https://aws.amazon.com/) 
+* Download and install the AWS Toolkit for Visual Studio, you can download the AWS toolkit from this [link](https://aws.amazon.com/visualstudio/).
+* The Toolkit can be installed from Tools/Extension and updates options in Visual Studio. 
 
-Create an AWS Lambda function to convert HTML to PDF and publish it to AWS.
-Invoke the AWS Lambda function in your main application using AWS SDKs.
+## Refer to the following steps to convert HTML to PDF in AWS Lambda
 
-## Steps to convert HTML to PDF in AWS Lambda:
+* Create an AWS Lambda function to convert HTML to PDF and publish it to AWS.
+* Invoke the AWS Lambda function in your main application using AWS SDKs.
 
-Create a new AWS Lambda project as follows:
+## Steps to convert HTML to PDF in AWS Lambda
+
+1. Create a new AWS Lambda project as follows:
 ![Convert HTMLToPDF AWS Step1](htmlconversion_images/AWS1.png) 
  
-Create a project name and select location.
+2. Create a project name and select location.
```

Comment:
In configuration window, name the project and select <italic>Create.

Reply:
Changed the above content